libs/net/dal/Migrations/20230707172534_1.0.42.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1042 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<string>(
```

Comment:
You need to update the `ContentConfiguration` and set the max length of the column.  This will require you to update your DB migration 1.0.42.  To do this you will need to do the following in the `/libs/net` folder.
`make db-rollback n=1.0.41`
`make db-remove`
`make db-add n=1.0.42`
`make db-update`